.travis.yml

```diff
@@ -9,8 +9,10 @@ scala:
   - 2.11.11
 
 script:
-  - "sbt clean coverage test scripted coverageReport coverageAggregate scalafmt headerCheck"
```

Comment:
Why separate this? the init time of sbt is a bit of a waste

Reply:
1. Once all the dependencies are cached, the init does not take so much time anymore. So start sbt twice maybe loses 10-20 seconds.
2. The tests take +- 25 minutes to run. Very annoying when these fail at the last moment because of formatting or header issues. That wasted 25 minutes.
So I think separating them saves a lot of time in the long run.